floris/simulation/turbine/operation_models.py

```diff
@@ -180,8 +180,8 @@ class CosineLossTurbine(BaseOperationModel):
     """
     Static class defining an actuator disk turbine model that may be misaligned with the flow.
     Nonzero tilt and yaw angles are handled via cosine relationships, with the power lost to yawing
-    defined by the pP exponent. This turbine submodel is the default, and matches the turbine
-    model in FLORIS v3.
+    defined by the cosine_loss_exponent_yaw. This turbine submodel is the default, and matches the
```

Comment:
The way this sentence reads is unclear. I suggest either rephrasing to plane english such as "defined the the corresponding cosine loss exponent for yaw and tilt".